_includes/sections/vpn.html

```diff
@@ -24,10 +24,12 @@ <h5><span class="badge badge-success">406+ Servers</span></h5>
     <h5><span class="badge badge-success">WireGuard Support</span></h5>
     <p>In addition to standard OpenVPN connections, Mullvad supports WireGuard. WireGuard is an experimental protocol with theoretically better security and higher reliability, although it is not currently recommended for production use.</p>
     <h5><span class="badge badge-success">Independently Audited</span></h5>
-    <p>Mullvad's VPN clients have been audited by Cure53 and Assured AB in a pentest report <a href="https://cure53.de/pentest-report_mullvad_v2.pdf">published at cure53.de</a>. The security researchers concluded:</p>
+    <p>Mullvad's VPN clients have been audited by Cure53 and Assured AB in a pentest report <a href="https://cure53.de/pentest-report_mullvad_v2.pdf">published at cure53.de</a> and . The security researchers concluded:</p>
     <blockquote class="blockquote">
       <p class="mb-0">...Cure53 and Assured AB are happy with the results of the audit and the software leaves an overall positive impression. With security dedication of the in-house team at the Mullvad VPN compound, the testers have no doubts about the project being on the right track from a security standpoint.</p>
     </blockquote>
+    <h5><span class="badge badge-success">Open source clients</span></h5>
+    <p>Mullvad provides the source code to their clients on Github at <a href="https://github.com/mullvad/mullvadvpn-app">mullvad/mullvadvpn-app</a>.</p>
```

Comment:
```suggestion
    <p>Mullvad provides the source code to their clients <a href="https://github.com/mullvad/mullvadvpn-app">on Github</a>.</p>
```

I think listing the path is ugly and it's also inconsistent with ProtonVPN below.

Reply:
Yeah you're probably right. When I wrote the one down below, I realized they keep it all in one repo.